### ISSUE TITLE
Destroyable containers

### ISSUE THREAD

**Issue by HAQ20** [Patbox/UniversalShops#31]
Is there any way to make the cointainer unbreakable. Other players can destroy your storage block and take all your stock, is there any way to combat that? Perhaps other mods?

**Comment by arnokeesman** [comment]
they can also just take from the storage containers
mod description suggests [HTM](https://github.com/QuiltServerTools/HeyThatsMine)

**Comment by HAQ20** [comment]
Thanks. I had a key that locked the chests but no way to prevent breaking. (also I thout the link wasn't a mod becouse of the short name so its totaly on me)

**Comment by HY916-cn** [comment]
> 谢谢。我有一把可以锁住箱子的钥匙，但没有办法防止被破坏。（另外，我认为这个链接不是模组，因为名字很短，所以完全是我的错）

modrinth.com/mod/htm

**Comment by Fabriziooo1** [comment]
Sadly, HTM doesn't have a version made for 1.21.1 on Fabric...

**Comment by HY916-cn** [comment]
> Sadly, HTM doesn't have a version made for 1.21.1 on Fabric...

1.21 and 1.21.1 is same, only fix there bugs , you can use 1.21 Fabric , here is the download link : https://cdn.modrinth.com/data/IEPAK5x6/versions/2tO12wyr/htm-1.1.13.jar

**Comment by Fabriziooo1** [comment]
> > Sadly, HTM doesn't have a version made for 1.21.1 on Fabric...
> 
> 1.21 and 1.21.1 is same, only fix there bugs , you can use 1.21 Fabric , here is the download link : https://cdn.modrinth.com/data/IEPAK5x6/versions/2tO12wyr/htm-1.1.13.jar

Yes i already tried it, the game launch, commands are displayed, but they don't work, i made a chest private and my friend still could open it...
